docs/src/modules/components/MuiBaseDeprecation.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import Alert from '@mui/material/Alert';
+import Link from '@mui/material/Link';
+
+export default function MuiBaseDeprecation(props: {
+  newComponentName?: string;
+  newComponentUrl?: string;
+}) {
+  if (props.newComponentUrl && props.newComponentName) {
+    return (
+      <Alert severity="warning">
```

Comment:
I think this warrants a big scary red callout to make sure people don't ignore it

```suggestion
      <Alert severity="error">
```